js/ui/select_box.js

```diff
@@ -384,15 +384,13 @@ var SelectBox = DropDownList.inherit({
     },
 
     _renderInputValue: function() {
-        this._renderInputValueAsync = function() {
-            this._renderTooltip();
-            this._renderInputValueImpl();
-            this._refreshSelected();
-        };
+        return this.callBase().always(this._renderInputValueAsync.bind(this));
+    },
 
-        return this.callBase().always((function() {
-            this._renderInputValueAsync();
-        }).bind(this));
+    _renderInputValueAsync: function() {
```

Comment:
This method should be defined dynamically because it will be cleared on dispose to solve problems with deferred loading